x/exchange/client/cli/query_setup.go

```diff
@@ -321,7 +321,7 @@ func MakeQueryGetCommitment(_ client.Context, flagSet *pflag.FlagSet, _ []string
 // SetupCmdQueryGetAccountCommitments adds all the flags needed for MakeQueryGetAccountCommitments.
 func SetupCmdQueryGetAccountCommitments(cmd *cobra.Command) {
 	cmd.Flags().String(FlagAccount, "", "The account's address")
-
+	cmd.Flags().String("asset", "", "Optional asset denom to filter commitments")
 	AddUseArgs(cmd,
 		fmt.Sprintf("{<account>|--%s <account>}", FlagAccount),
 	)
```

Comment:
You should add a new entry for the newly added flag.

```suggestion
	AddUseArgs(cmd,
		fmt.Sprintf("{<account>|--%s <account>}", FlagAccount),
		fmt.Sprintf("[--%s <denom>]", FlagDenom),
	)
```